10_4_3_Reading_Charts_Graphs_Powerpoints.ipynb

```diff
@@ -145,7 +145,7 @@
     "        {\n",
     "            \"role\": 'user',\n",
     "            \"content\": [\n",
-    "                {\"text\": \"What's in this image? Answer in a single sentence.\"},\n",
+    "                {\"text\": f\"{question}\"},\n",
```

Comment:
just letting you know if you gonna pass in just the quest u dont need the f-string ,you can pass in this way:

`{"text": {question}}`